Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG GOARCH="amd64"
```

Comment:
we want to build only amd64?

Reply:
I just make this as a stopgap, feel free to improve it